lib/index.js

```diff
@@ -71,7 +79,7 @@ module.exports = postcss.plugin('alter-color', options => {
         cssTree.walk(parsedValue, {
           visit: 'Function',
           enter(node) {
-            applyColorFunction(node, initialColor, finalColor);
+            applyColorFunction(node, initialColor, finalColor, options.alphaChannel);
```

Comment:
As I understand, `options.alphaChannel` means that you don’t want to modify alpha channel. Right?

I suggest name the options like `preserveAlphaChannel`.

Reply:
Okey